extensions/mssql/src/mssql.d.ts

```diff
@@ -42,22 +45,9 @@ declare module 'mssql' {
 
 		readonly sqlProjects: ISqlProjectsService;
 
-		readonly sqlAssessment: ISqlAssessmentService;
-
 		readonly azureBlob: IAzureBlobService;
 	}
 
-	/**
-	 * A browser supporting actions over the object explorer connections provided by this extension.
-	 * Currently this is the
-	 */
-	export interface MssqlObjectExplorerBrowser {
```

Comment:
I also cleaned up these, which were left over from the BDC removal and not being used anymore.